packages/create-react-native-library/src/index.ts

```diff
@@ -632,7 +635,7 @@ async function create(argv: yargs.Arguments<any>) {
     : 'legacy';
 
   const example =
-    hasExample && !local ? (type === 'library' ? 'expo' : 'native') : 'none';
+    hasExample && !local ? (type === 'library' ? 'expo' : hasExample) : null;
```

Comment:
Does `test-app` have web support? Wondering if we can use it for `library` type

Reply:
https://github.com/microsoft/react-native-test-app/issues/812#issuecomment-1071134434